cmake/environmentchecks.cmake

```diff
@@ -121,7 +121,17 @@ else()
 endif()
 
 if(CLANG_FORMAT_EXE AND GIT_EXE)
-  set(CLANG_FORMAT_TESTING_AVAILABLE TRUE CACHE INTERNAL "Executables required to run the ClangFormat test are available.")
+  set(CLANG_FORMAT_VERSION_NEEDED 11)
```

Comment:
I would rename this variable to `CLANG_FORMAT_VERSION_MINIMUM`.

Reply:
Agreed, done in commit 38011d81d.